test/unit/BPool.t.sol

```diff
@@ -1362,18 +1362,28 @@ contract BPool_Unit_SwapExactAmountOut is BasePoolTest {
     bPool.swapExactAmountOut(tokenIn, _maxAmountIn, tokenOut, _fuzz.tokenAmountOut, _maxPrice);
   }
 
-  function test_Revert_NotBoundTokenIn(address _tokenIn) public {
+  function test_Revert_NotBoundTokenIn(
+    address _tokenIn,
+    SwapExactAmountOut_FuzzScenario memory _fuzz,
+    uint256 _maxAmountIn,
+    uint256 _maxPrice
+  ) public happyPath(_fuzz) {
     vm.assume(_tokenIn != VM_ADDRESS);
 
     vm.expectRevert('ERR_NOT_BOUND');
-    bPool.swapExactAmountOut(_tokenIn, 0, tokenOut, 0, 0);
+    bPool.swapExactAmountOut(_tokenIn, _maxAmountIn, tokenOut, _fuzz.tokenAmountOut, _maxPrice);
```

Comment:
Could you fuzz all the function's general parameters within the `SwapExactAmountOut_FuzzScenario` struct?